src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,666 @@
++++
+title = "User Guide"
+date = 2020-05-04T19:01:14+05:30
+weight = 15
++++
+
+## Overview
+
+This guide is intended for users who have a Submariner environment set up and want to verify the installation and learn more about how to
+use Submariner and the main capabilities it provides. This guide assumes that there are two Kubernetes clusters,
+**cluster-2** and **cluster-3**, forming a clusterset, and that the Broker is deployed into a separate cluster **cluster-1**.
+
+{{% notice note %}}
+This guides focuses on a non-Globalnet Submariner deployment.
+{{% /notice %}}
+
+### 1. Validate the Installation
+
+#### Broker
+
+The Broker facilitates the exchange of metadata information between the connected clusters, enabling them to discover one another.
+The Broker consists of a set of Custom Resource Definitions (CRDs) only; there are no Pods or Services deployed with it.
+
+You can use this command to validate that the Broker namespace has been created in the Broker cluster:
```

Comment:
Maybe add `kubectl config use-context` cluster-1? but the description makes it clear which cluster to use